pages/community.ca.md

```diff
@@ -15,18 +15,18 @@ a obtenir suport de la comunitat.
 
 Trobareu molts materials útils a la nostra wiki, però també us animem a millorar-la.
 
-  * <http://wiki.mate-desktop.org>
+  * <https://wiki.mate-desktop.org>
 
 ## Fòrums
 
 No utilitzeu el correu electrònic, també ho aconseguirem. Viniu i uniu-vos als
 fòrums de la comunitat.
 
   * [Fòrums d'Arch Linux](https://bbs.archlinux.org/)
-  * [Fòrums d'usuaris de Debian](http://forums.debian.net/)
-  * [Fòrums de Fedora](http://fedoraforum.org/)
+  * [Fòrums d'usuaris de Debian](https://forums.debian.net/)
```

Comment:
Any idea what should i do with http://forums.debian.net? No https.

Reply:
test.sh
```shell
#!/bin/bash
url=$1
if ! curl --output /dev/null --silent --head --connect-timeout 3 --fail "$url"; then
  echo "URL does not exist: $url"
fi
```
Test
```shell
grep -hoPrs --include=\*.md '\(http.*?\)' | tr -d '()' | sort -u | xargs -i --max-procs=10 bash -c "./test.sh {}"
```
Output
```
URL does not exist: https://download.opensuse.org/repositories/X11:/MATE:/Current/openSUSE_12.3/X11:MATE:Current.repo
URL does not exist: https://galculator.sourceforge.net/
URL does not exist: https://alioth.debian.org/projects/pkg-mate
URL does not exist: https://forums.debian.net/
URL does not exist: https://forum_archive.mate-desktop.com
URL does not exist: https://github.com/NiceandGently
URL does not exist: https://users.physik.fu-berlin.de/~glaubitz/
URL does not exist: https://wiki.mate-desktop.com/board
URL does not exist: https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s10.html
URL does not exist: https://wiki.mate-desktop.com/download
URL does not exist: https://wiki.mate-desktop.org
URL does not exist: https://wiki.mate-desktop.org/download
URL does not exist: https://wiki.mate-desktop.org/dev-doc
URL does not exist: https://wiki.mate-desktop.org/download/
URL does not exist: https://wiki.mate-desktop.org/download#fedora
URL does not exist: https://wiki.mate-desktop.org/irc
URL does not exist: https://wiki.mate-desktop.org/gsoc:2014
URL does not exist: https://wiki.mate-desktop.org/roadmap
URL does not exist: https://ml.mate-desktop.org/listinfo/
URL does not exist: https://ml.mate-desktop.org/pipermail/mate-dev/2014-February/000228.html
URL does not exist: https://www.google-melange.com/
URL does not exist: https://www.google-melange.com/gsoc/document/show/gsoc_program/google/gsoc2014/help_page
URL does not exist: https://www.bytemark.co.uk/r/ubuntu-mate/
URL does not exist: https://www.ltsp.org/
URL does not exist: https://www.linuxvoice.com/
URL does not exist: https://www.reddit.com/r/MATEDesktop
URL does not exist: https://www.xnoise-media-player.com/
```

Reply:
There is no official support forum in Catalan for Fedora and Debian right now.
```patch
diff --git a/pages/community.ca.md b/pages/community.ca.md
index 9a29e6e..5324db8 100644
--- a/pages/community.ca.md
+++ b/pages/community.ca.md
@@ -23,8 +23,8 @@ No utilitzeu el correu electrònic, també ho aconseguirem. Viniu i uniu-vos als
 fòrums de la comunitat.
 
   * [Fòrums d'Arch Linux](https://bbs.archlinux.org/)
-  * [Fòrums d'usuaris de Debian](https://forums.debian.net/)
-  * [Fòrums de Fedora](https://fedoraforum.org/)
+  * [Fòrums d'usuaris de Debian](http://forums.debian.net/)
+  * [Fòrums de Fedora](https://ask.fedoraproject.org/)
   * [Fòrums de Gentoo](https://forums.gentoo.org/)
   * [Fòrums de Linux Mint](https://forums.linuxmint.com/)
   * [Fòrums d'Ubuntu MATE](https://ubuntu-mate.community)
```

Reply:
Should i keep http link?

Reply:
That fedora or debian forum  don't support Catalan language isn't our problem.
Link is working.

Reply:
I will use http://forums.debian.net/ again

Reply:
if `http` works and `https` does't work, I vote to keep the `http` link

Reply:
Done.
@rbuj 
I can add https://ask.fedoraproject.org/ to catalan md. But this site isnt a forum, ihmo.
Maybe you fix that later?